modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -458,13 +459,20 @@ export default {
       }
     });
     apos.bus.$on('apos-refreshing', this.onAposRefreshing);
+    this.$refs.editor.addEventListener('cut', this.ignoreEvent)
```

Comment:
not sure why we need these ones

Reply:
because we don't want to apostrophe-cut the rich text widget we're typing in and using cut in the normal, textual way